assets/src/scss/layout/_country-selector.scss

```diff
@@ -0,0 +1,189 @@
+
+#country-selector {
+  _-- {
+    color: inherit;
+  }
+  font-size: $font-size-sm;
+  margin-bottom: 24px;
+  text-align: start;
+  width: 100%;
+
+  a _-- {
+    color: inherit;
+  }
+
+  button:hover {
+    text-decoration: var(--link--hover--text-decoration, underline);
+  }
+
+  button:focus {
+    border-radius: var(--link--focus--border-radius, 4px);
+    outline: var(--link--focus--border, 2px solid transparentize($white, 0.6));
+    outline-offset: 0;
+  }
+
+  button:focus:not(:focus-visible) {
+    outline: 0;
+  }
+
+  .control-toggle {
+    align-self: flex-start;
+    background: transparent;
+    border: none;
+    color: var(--site-footer--color, $white);
+    cursor: pointer;
+    text-align: start;
+    white-space: nowrap;
+    width: 100%;
+
+    &::before,
+    &::after {
+      background-color: var(--site-footer--color, $white);
+      content: "";
+      display: inline-block;
+      width: 16px;
+    }
+
+    &::before {
+      height: 20px;
+      margin-inline-end: 8px;
+      mask: url("../../images/gp-g-logo.png") 0 0/16px 20px no-repeat;
+      vertical-align: middle;
+    }
+
+    &::after {
+      height: 12px;
+      margin-inline-start: 8px;
+      mask: url("../../images/chevron.svg") 0 0/16px 12px no-repeat;
+      transform: rotate(0.25turn);
+      transition: transform 300ms cubic-bezier(.86, 0, .07, 1);
+    }
+  }
+
+  .countries-list {
```

Comment:
> (if there is any)

PTSD from trying to not conflict with existing rules when implementing this and the navbar :grin:  Having everything in one specific selector made it easy to not have side effects on other existing themes/components and make 2 codebases live together. But I see the other side of the coin, I'll take stuff out of the main selector.

Reply:
Just to have some reference on the PR from things we discussed elsewhere:

Because the new and old CSS were conflicting, which forced us to complexify the new CSS so that both could be loaded together without affecting each other, I gave the suggestion I made earlier in this PR a shot. I [split off the country selector related code to 2 separate entrypoints](https://github.com/greenpeace/planet4-master-theme/pull/1521), and conditionally made it use either the old or the new version. That way it never has both versions loaded at the same time.